projects/1.16/assets/more-jellyfish/more_jellyfish/lang/zh_cn.json

```
@@ -0,0 +1,104 @@
+{
+
+  "block.more_jellyfish.deep_coralstone": "深海珊瑚石",
+  "block.more_jellyfish.deep_coralstone_wall": "深海珊瑚石墙",
+  "block.more_jellyfish.deep_coralstone_stairs": "深海珊瑚石楼梯",
+  "block.more_jellyfish.deep_coralstone_slab": "深海珊瑚石台阶",
+  "block.more_jellyfish.deep_coralstone_vertical_slab": "深海珊瑚石竖台阶",
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
  "block.more_jellyfish.deep_coralstone_vertical_slab": "深海珊瑚石竖直台阶",
```
下方同理。

---

已修正